### PR TITLE
UI blackjack

### PR DESCRIPTION
**Pull request by BrassoiCatalin**
Started implementing the UI for Blackjack game.

### CODE REVIEW

File: SevenSlots/SevenSlots/SevenSlots.Android/SevenSlots.Android.csproj [hunk 1]
```diff
@@ -164,5 +164,161 @@
...
+    <AndroidResource Include="Resources\drawable\blackjack_10_spade.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\drawable\blackjack_11_spade.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\drawable\blackjack_12_spade.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\drawable\blackjack_13_spade.png" />
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots.Android/SevenSlots.Android.csproj:321]: Too much space in andooid resource tag

File: SevenSlots/SevenSlots/SevenSlots/View/App.xaml [hunk 2]
```diff
@@ -27,6 +27,24 @@
...
+                <Setter Property="WidthRequest" Value="50"/>
+                <Setter Property="FontSize" Value="20"/>
+                <Setter Property="TextColor" Value="#a62523"/>
+                <Setter Property="Background" Value="#710824"/>
+                <Setter Property="CornerRadius" Value="100"/>
+            </Style>
+            <Style x:Key="BlackjackLabel" TargetType="Label">
+                <Setter Property="FontSize" Value="Medium"/>
+                <Setter Property="TextColor" Value="#710824"/>
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/View/App.xaml]: Extra new line

File: SevenSlots/SevenSlots/SevenSlots.Android/SevenSlots.Android.csproj (hunk 1 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots.Android/SevenSlots.Android.csproj:321]: Those are auto generated by Visual Studio, not by me.

File: SevenSlots/SevenSlots/SevenSlots/View/App.xaml (hunk 2 above)
Comment (BrassoiCatalin) [reply SevenSlots/SevenSlots/SevenSlots/View/App.xaml]: Will fix it.

**Review by Sorin37 — APPROVED** [review]

**Pull request merged**